Changelog.md

```diff
@@ -4,7 +4,7 @@ Language Features:
 
 
 Compiler Features:
-
+* ethdebug: Experimental support for instructions and source locations under EOF.
```

Comment:
```suggestion

* ethdebug: Experimental support for instructions and source locations under EOF.
```

Reply:
Uhh. Happy to fix this - but why? It's not like there's a blank line between bullet point list and headline anywhere else.

Reply:
Ah, my bad, should have been the other way around - an extra empty line below :)

> but why?

Two empty lines between sections.

Maybe we should just change the convention to a single line? No one seems to be able to keep it straight or see it in the diff and it gets changed by PRs back and forth. I usually ignore it but I got annoyed enough by it seeing it again to suggest a correction :)

Reply:
Haha yeah, that explains it. Might've also been my pre-coffee confusion that I didn't realize you meant the blank line below. I am absolutely fine with changing it to one line in any case! :)